api/src/main/java/com/messagebird/objects/conversations/MessageParam.java

```diff
@@ -71,17 +81,30 @@ public void setImage(Media image) {
 
     public void setVideo(Media video) { this.video = video; }
 
+    public String getExpirationTime() {
+        return expirationTime;
+    }
+
+    public void setExpirationTime(String expirationTime) {
+        if (expirationTime == null || expirationTime.isEmpty()) {
```

Comment:
same here if we have StringUtils.isEmpty, it is more readible

Reply:
updated, thanks